src/test/java/software/amazon/awssdk/crt/test/CrtTestFixture.java

```diff
@@ -230,6 +236,7 @@ public static void setupOnce() {
     /* The setup function will be run before every test */
     @Before
     public void setup() {
+        System.out.println("[TEST START] " + testName.getMethodName());
```

Comment:
https://github.com/awslabs/aws-crt-java/actions/runs/19476392289/job/55739151489?pr=938#step:4:2041
Do we really want all of those print out in the log? Is there an example about this helping debugging?

Reply:
There are times when tests fail but there is no record of which test from a class of tests was the one that failed. I've seen it most when tests time out and logs need to be truncated.